shared-lib/spiutils/src/cmds.rs

```diff
@@ -0,0 +1,86 @@
+/// SPI flash op codes
+#[repr(u8)]
+#[allow(dead_code)]
```

Comment:
`#[allow(dead_code)]` shouldn't be necessary here.

Reply:
Ah yes, good catch, thank you.